transform.test.ts

```diff
@@ -50,6 +50,22 @@ const testCases: TestCase[] = [
       }
     `,
   },
+  {
+    input: `
+      import React from 'react';
+
+      type Props2 = { id: number };
+      const MyComponent: React.FunctionComponent<Props2> = (props) => {
+        return <span>{props.id}</span>
+      }`,
+    output: `
+      import React from 'react';
+
+      type Props2 = { id: number };
+      const MyComponent = (props: Props2) => {
+        return <span>{props.id}</span>
+      }`,
+  },
```

Comment:
Can we add a scenario where using `FunctionComponent` without props? Something that shows how this

```js
const MyComponent: React.FunctionComponent = () => <span>
```

is converted properly, for the sake of having more scenarios tested